tests/lambda_endpoints/test_query_api.py

```diff
@@ -22,7 +22,7 @@ def _populate_test_data(session):
         "version": "v001",
         "extension": "pkts",
         "ingestion_date": datetime.datetime.strptime(
-            "2025-11-07 10:13:12+00:00", "%Y-%m-%d %H:%M:%S%z"
+            "20251107 10:13:12+00:00", "%Y%m%d %H:%M:%S%z"
```

Comment:
Since this test setup function is writing to the database, you will want that in the same date format as this [function](https://github.com/IMAP-Science-Operations-Center/sds-data-manager/blob/dev/sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/indexer.py#L22) since this is the date format we write  to the database on file ingestion. 
```suggestion
            "2025-11-07 10:13:12+00:00", "%Y-%m-%d %H:%M:%S%z"
```